icees_api/handlers.py

```diff
@@ -198,7 +198,7 @@ def feature_association(
     cohort_meta = sql.get_features_by_id(conn, table, cohort_id)
 
     if cohort_meta is None:
-        return_value = "Input cohort_id invalid. Please try again."
+        raise HTTPException(status_code=422, detail="Input cohort_id invalid. Please try again.")
```

Comment:
I don't know that I agree with the 422 error code used in all these exceptions. Unprocessable Entity tells the user that their input is malformed, rather than we just didn't find any results. In my mind, no results is a valid response and should instead be a 200 status code. @karafecho does this make sense? I remember you saying these used to return errors?